test/geometry_tests.cpp

```diff
@@ -162,6 +204,48 @@ TEST(Point2DPlanning_Griz_Test, bresenham_conversion_octant1_test) {
 
 }
 
+TEST(Point2DPlanning_Griz_Test, raytraceLine_conversion_pointTest) {
+	Cell startPix = {.x = 0, .y = 0};
+	Cell endPix = {.x = 0, .y = 0};
+	auto pixels = raytraceLine_conversion(startPix, endPix);
+
+	// printPixels(pixels);
+
+	ASSERT_EQ(pixels.size(), 1);	
+}
+
+TEST(Point2DPlanning_Griz_Test, raytraceLine_conversion_horz_line_test) {
+	Cell startPix = {.x = 0, .y = 0};
```

Comment:
Can you adopt the BDD style of commenting and organizing your tests?
That would look like
```
// GIVEN some precondition
... set up the preconditions
// WHEN the action or event you're testing
... do the thing
// THEN the expected result
... EXPECT_
```

For this test it would look like
```
// GIVEN horizontal pixel coordinates
const Cell start = {0, 0};
const Cell end = {5, 0};

// WHEN computing the raytrace
const auto pixels = raytraceLine_conversion(start, end);

// THEN we should get a horizontal line.
EXPECT_EQ(pixels.size(), 6);	
EXPECT_EQ(allXOrYCoorSame(1,pixels), true);
```

The second part should be a separate test. Though, this screams to be a parameterized test
https://github.com/google/googletest/blob/master/googletest/docs/advanced.md#value-parameterized-tests

Reply:
I'll do this after you look after everything else.